project/src/components/genres list/genres-list.tsx

```diff
@@ -10,7 +9,7 @@ type Props = {
 function GenresList({genres, activeGenre}: Props): JSX.Element {
   const dispatch = useAppDispatch();
 
-  const handleChangeActiveGenre = (genre: string) => {
+  const handleGenreChange = (genre: string) => {
```

Comment:
The filter value is used only in one component, so you don't need to move it to the store

Reply:
Leave it as state